environment.yml

```
@@ -1,247 +1,22 @@
-      - nbclassic==0.5.5
-      - nbclient==0.7.3
-      - nbconvert==7.3.0
-      - nbformat==5.8.0
-      - nest-asyncio==1.5.6
-      - networkx==3.1
-      - nodeenv==1.8.0
-      - notebook==6.5.4
-      - notebook-shim==0.2.2
-      - numba==0.56.4
-      - numpy==1.23.0
-      - packaging==23.0
-      - pandas==2.0.0
-      - pandocfilters==1.5.0
-      - parso==0.8.3
-      - partd==1.4.0
-      - patsy==0.5.3
-      - pexpect==4.8.0
-      - pickleshare==0.7.5
-      - pillow==9.5.0
-      - platformdirs==3.2.0
-      - plotly==5.14.1
-      - pluggy==1.0.0
-      - pre-commit==3.3.1
-      - prometheus-client==0.16.0
-      - prompt-toolkit==3.0.38
-      - psutil==5.9.4
-      - ptyprocess==0.7.0
-      - pure-eval==0.2.2
-      - pyarrow==11.0.0
-      - pycodestyle==2.10.0
-      - pycparser==2.21
-      - pydocstyle==6.3.0
-      - pyflakes==3.0.1
-      - pygments==2.14.0
-      - pylint==2.17.2
-      - pynisher==0.6.4
-      - pynndescent==0.5.8
-      - pyparsing==3.0.9
-      - pyrfr==0.8.3
-      - pyrsistent==0.19.3
-      - python-dateutil==2.8.2
-      - python-dotenv==1.0.0
-      - python-json-logger==2.0.7
-      - python-lsp-jsonrpc==1.0.0
-      - python-lsp-server==1.7.2
-      - pytoolconfig==1.2.5
-      - pytz==2023.3
-      - pywavelets==1.4.1
-      - pyyaml==6.0
-      - pyzmq==25.0.2
-      - requests==2.28.2
-      - rfc3339-validator==0.1.4
-      - rfc3986-validator==0.1.1
-      - rope==1.7.0
-      - salib==1.4.7
-      - scikit-image==0.20.0
-      - scikit-learn==0.24.2
-      - scikit-posthocs==0.7.0
-      - scipy==1.9.1
-      - seaborn==0.12.2
-      - send2trash==1.8.0
-      - shap==0.41.0
-      - six==1.16.0
-      - skope-rules==1.0.1
-      - slicer==0.0.7
-      - smac==1.2
-      - sniffio==1.3.0
-      - snowballstemmer==2.2.0
-      - sortedcontainers==2.4.0
-      - soupsieve==2.4
-      - stack-data==0.6.2
-      - statsmodels==0.13.5
-      - tblib==1.7.0
-      - tenacity==8.2.2
-      - terminado==0.17.1
-      - threadpoolctl==3.1.0
-      - tifffile==2023.4.12
-      - tinycss2==1.2.1
-      - toml==0.10.2
-      - tomli==2.0.1
-      - tomlkit==0.11.7
-      - toolz==0.12.0
-      - tornado==6.2
-      - tqdm==4.65.0
-      - traitlets==5.9.0
-      - treeinterpreter==0.2.3
-      - typing-extensions==4.5.0
-      - tzdata==2023.3
-      - ujson==5.7.0
-      - umap-learn==0.5.3
-      - uri-template==1.2.0
-      - urllib3==1.26.15
-      - virtualenv==20.23.0
-      - wcwidth==0.2.6
-      - webcolors==1.13
-      - webencodings==0.5.1
-      - websocket-client==1.5.1
-      - werkzeug==2.3.4
-      - whatthepatch==1.0.4
-      - wrapt==1.15.0
-      - xgboost==1.7.5
-      - y-py==0.5.9
-      - yapf==0.32.0
-      - ypy-websocket==0.8.2
-      - zict==2.2.0
-      - zipp==3.15.0
-      - zope-event==4.6
-      - zope-interface==6.0
-prefix: /home/camo/anaconda3/envs/nf1_analysis
+  - conda-forge::python=3.9
+  - umap-learn
+  - scikit-posthocs
+  - pandas
```

pandas 2.0 is out now. To avoid code decay I recommend adding a version limit on this
```suggestion
  - pandas<2.0
```

---

Sounds good, changed!